core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelScan_Range.hpp

```diff
@@ -76,10 +77,12 @@ class ParallelScan<FunctorType, Kokkos::RangePolicy<Traits...>,
     idx_type nteams           = n_chunks > 512 ? 512 : n_chunks;
     idx_type team_size        = 128;
 
-    FunctorType a_functor(m_functor);
-#pragma omp target teams distribute map(to : a_functor) num_teams(nteams)
+    auto a_functor_reducer = m_functor_reducer;
+#pragma omp target teams distribute map(to \
+                                        : a_functor_reducer) num_teams(nteams)
```

Comment:
We should be able to get rid of this indirection of `a_functor_reducer` and directly pass `m_functor_reducer` . But maybe that can be done separately.